websites/B/Bounce/dist/metadata.json

```diff
@@ -11,12 +11,14 @@
     "ga_IE": "Is stáisiún ceoil idirnáisiúnta buailte é Bounce Radio a chuireann ceol buailte ar fáil do gach duine. Déanta ag gamers do gamers."
   },
   "url": [
-    "wearebounce.net",
-    "www.wearebounce.net"
+    "itsbounce.net",
+    "itsbounce.com",
+    "itsbounce.uk",
+    "wearebounce.co.uk"
   ],
-  "version": "2.5.3",
-  "logo": "https://boun.cc/cdn/noshadow.png",
-  "thumbnail": "https://i.imgur.com/XmJTiZY.png",
+  "version": "3.0.0",
+  "logo": "https://www.itsbounce.net/branding/avatar.png",
```

Comment:
only imgur 😉

Reply:
we are allowed to use our own cdn, I know the rules xoxoxoxoxoxoxoxoxoxoxoxooxoxoxxoxoxoxoxoxoxoxoxoxoxoxooxoxox

Reply:
![image](https://user-images.githubusercontent.com/34792838/110222910-33144680-7ed6-11eb-92d2-563916bb1988.png)
Has been changed a while back